README.md

```diff
@@ -75,28 +94,28 @@ class DemoOne extends React.Component {
   render() {
     const trap = this.state.activeTrap
       ? <FocusTrap
+          component="div"
```

Comment:
@dakeyras7 This is a mistake, right? — there's no `component` prop added? I've merged but thought I'd clean this up. Please let me know if it my assumption is wrong.

Reply:
Yeah, my bad. I think this is a remnant from my previous suggestion. Good catch!

You can just remove this line.